### PR TITLE
Add <<= and >>= operators

### PR DESCRIPTION
**Pull request by TiagoCavalcante**

### CODE REVIEW

**Comment by ThakeeNathees** [comment]
Seems like you've used your last branch as an origin for this branch. To solve the merge conflicts you need to rebase your branch with the master 

Here is how to do it, in case if you don't know already
```
git remote add upstream https://github.com/ThakeeNathees/pocketlang.git
git fetch upstream master
git rebase upstream/master
```
At this point, you have to fix the conflicts using [git conflict marker](https://wincent.com/wiki/Understanding_Git_conflict_markers) once you've done.
```
git add .
git rebase --continue
```

--------

Next time create a new branch from the `master`.
```
git checkout master
git checkout -b a-new-feature
```

**Comment by TiagoCavalcante** [comment]
@ThakeeNathees yes, I was without time to rebase, but now it's done.

File: src/pk_compiler.c [hunk 1]
```diff
@@ -1880,6 +1894,8 @@ static void emitAssignment(Compiler* compiler, TokenType assignment) {
     case TK_OREQ: emitOpcode(compiler, OP_BIT_OR); break;
     case TK_XOREQ: emitOpcode(compiler, OP_BIT_XOR); break;
     case TK_MODEQ: emitOpcode(compiler, OP_MOD); break;
```
Comment (ThakeeNathees) [inline src/pk_compiler.c]: Move this `TK_MODEQ` above to match the order in the `Tokens` enums, (I've forgotten to update it after the `%=` PR)

File: src/pk_compiler.c [hunk 2]
```diff
@@ -991,6 +1000,9 @@ static bool matchAssignment(Compiler* compiler) {
   if (match(compiler, TK_ANDEQ)) return true;
   if (match(compiler, TK_OREQ)) return true;
   if (match(compiler, TK_XOREQ)) return true;
+  if (match(compiler, TK_MODEQ)) return true;
```
Comment (ThakeeNathees) [inline src/pk_compiler.c]: `TK_MODEQ` already handled above (after the `TK_DIVEQ`), remove it here.

**Review by ThakeeNathees — CHANGES_REQUESTED** [review]
Looks good to me. There is a small change I'd like, before merging this.

File: src/pk_compiler.c (hunk 2 above)
Comment (TiagoCavalcante) [reply src/pk_compiler.c]: Both done.

**Review by ThakeeNathees — APPROVED** [review]
Looks great, Thanks for opening this PR.

**Pull request merged**